jina/serve/streamer.py

```diff
@@ -221,3 +221,36 @@ def get_streamer():
     @staticmethod
     def _set_env_streamer_args(**kwargs):
         os.environ['JINA_STREAMER_ARGS'] = json.dumps(kwargs)
+
+
+class _ExecutorStreamer:
```

Comment:
@JoanFM  is it worth it to have an abstract client class common to both the post mixin and this class ?